docs/enterprise-setup/implementation-guide.md

```diff
@@ -204,82 +196,7 @@ Follow these instructions to add the Airbyte helm repository:
 2. Perform the repo indexing process, and ensure your helm repository is up-to-date by running `helm repo update`.
 3. You can then browse all charts uploaded to your repository by running `helm search repo airbyte`.
 
-### Step 2: Create your Enterprise License File
-
-1. Create a new `airbyte` directory. Inside, create an empty `airbyte.yml` file.
-
-2. Paste the following into your newly created `airbyte.yml` file:
-
-<details>
-<summary>Template airbyte.yml file</summary>
-
-```yaml
-webapp-url: # example: http://localhost:8080
-
-initial-user:
-  email:
-  first-name:
-  last-name:
-  username: # your existing Airbyte instance username
-  password: # your existing Airbyte instance password
-
-license-key: # license key provided by Airbyte team
-```
-
-</details>
-
-3. Fill in the contents of the `initial-user` block. The credentials grant an initial user with admin permissions. You should store these credentials in a secure location.
-
-4. Add your Airbyte Self-Managed Enterprise license key to your `airbyte.yml` in the `license-key` field.
-
-5. To enable SSO authentication, add [SSO auth details](/access-management/sso) to your `airbyte.yml` file.
```

Comment:
Alex, we should probably bring some of this verbiage with us, not sure where it goes though.

Reply:
Instead of telling people to paste a single block in the current step 2, you should break it out into multiple steps.

I also am not sure that the license key should be in the same Kubernetes secret as the rest of the infrastructure